sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/append_blob_client.hpp

```diff
@@ -119,8 +119,8 @@ namespace Azure { namespace Storage { namespace Blobs {
      * exists.
      *
      * @param options Optional parameters to execute this function.
-     * @return A CreateAppendBlobResult describing the newly created append blob. Null if the blob
-     * already exists.
+     * @return A CreateAppendBlobResult describing the newly created append blob.
+     * CreateAppendBlobResult.Created is false if the blob already exists.
```

Comment:
nit.
```suggestion
     * The field `CreateAppendBlobResult.Created` is false if the blob already exists.
```

Reply:
Does doxygen generated stuff support markdown syntax?

Reply:
Some of it.
At least the " ` " works.  Take look to some sample like context::ValueBase :
https://azuresdkdocs.blob.core.windows.net/$web/cpp/azure-core/1.0.0-beta.3/struct_azure_1_1_core_1_1_value_base.html

Something like:
![image](https://user-images.githubusercontent.com/24213737/104272630-698f9100-5452-11eb-859d-e1d10921757a.png)

Resolves to:
![image](https://user-images.githubusercontent.com/24213737/104272610-5da3cf00-5452-11eb-9be0-face7319bf5d.png)

Reply:
@vhvb1989 Good to know that. I think I'm going to update all docs.

Reply:
@vhvb1989 BTW, do you know how to reference another class, or member of another class? Reference means it's a link in the generated doc.

Reply:
There are a few other nice things we can do to improve the doc's looking.  I know this other one with the " \p " for args like

this:
![image](https://user-images.githubusercontent.com/24213737/104272911-ffc3b700-5452-11eb-878c-9882781c4ca8.png)

Resolves to this:
![image](https://user-images.githubusercontent.com/24213737/104272932-0a7e4c00-5453-11eb-9677-df92e92210b1.png)

The best guy for this is @ahsonkhan  :)

Reply:
>  do you know how to reference another class

I have seen others to use `#` for that. But I think that only works if the reference is globally known. Otherwise, maybe we need to add the full link like `#Azur::Core::Context` and not only `#Context`.  
I am not sure. I will wait to see if @ahsonkhan knows it already, otherwise, I can learn it some day and share hahah..  We have some broken references like:

![image](https://user-images.githubusercontent.com/24213737/104274728-5c74a100-5456-11eb-8502-53a0d609c153.png)

See how it just displays the " # "  .. The expectation would be a link.   We might even have an issue for this somewhere in the backlog :)

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/1327

Reply:
Just tested with full path and worked:

This:
![image](https://user-images.githubusercontent.com/24213737/104277732-32be7880-545c-11eb-8c09-21321f66cbcd.png)

Generates:
![image](https://user-images.githubusercontent.com/24213737/104277719-2d612e00-545c-11eb-80ba-b2325119b451.png)

Reply:
Btw, you can generate the documentation locally by doing:

```cpp
cmake -DBUILD_DOCUMENTATION=ON ..
cmake --build .
```

And in case you are running from a VM with linux using ssh, you can start an HTTP server from `azure-sdk-for-cpp/build/sdk` and browse to `localhost:8000` for instance, to see how each package's docs looks like

Reply:
> Just tested with full path and worked:

Great! That's what I expected :)

Reply:
> Btw, you can generate the documentation locally by doing:

We should add that to the README or contributing guide (currently, we only mention it in the cmake options table).